config/jobs/kubernetes/release/release-config.yaml

```
@@ -46,3 +46,17 @@ presubmits:
           args:
             - test
             - ./...
+  - name: pull-release-verify
+    always_run: false
```

If you want this to be optional I recommend:
```suggestion
    always_run: true
    optional: true
```

As written you'll have to manually `/test pull-release-verify` to trigger and it will add a required status context.

---

Awesome suggestion, I made the change.